### DISCUSSION TITLE
Number of search results

### DISCUSSION THREAD

**Discussion by jagjordi** [immich-app/immich#4417] [Q&A] [answered]
Is the number of search results limited?

It seems it is always the same number of outputs

**Comment by jrasm91** [comment]
Yeah it is 250 at the moment.

**Reply by jagjordi** [reply]
Is it possible to make it longer? Or that when you scroll to the end it queries 250 more?

**Reply by jrasm91** [reply, marked as answer]
It's complicated because search/typesense has some bigger issues we need to figure out first. We may migrate to `pgvector` and then come back to search enhancements afterwards.
